pages/index.tsx

```diff
@@ -35,7 +25,7 @@ export default function HomePage() {
   const [status, setStatus] = useState("");
 
   const { data } = useQuery(["submissions", currentPage, status], () =>
-    getData(currentPage, status)
+    submissionService.getSubmissions(currentPage, status, token)
```

Comment:
you can consume the token directly inside of the getSubmissions function. I find this a better approach because you no longer need to even know `token` exists in the react components